Sources/RSDKUtils/UnitTests/URLSessionMock.swift

```diff
@@ -0,0 +1,113 @@
+import Foundation
+
+public final class URLSessionMock: URLSession {
```

Comment:
I'm not sure about moving this class. It is only used by SenderSpec I think. I was thinking about deleting this class and using SwiftySessionable instead in SenderSpec. It is because I prefer mocking with protocols than mocking with swizziling.

Reply:
I agree that this is not an ideal method for mocking. I brought it here because it's used also in IAM tests